### PR TITLE
New Function Get-DbaDbObjectTrigger & Improve Find-DbaTrigger

### PR DESCRIPTION
**Pull request by ClaudioESSilva**
## Type of Change

 - [ ] Bug fix (non-breaking change, fixes #<enter issue number>)
 - [x] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [x] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
new function `Get-DbaDbObjectTrigger` that search for triggers on object level. This means Tables and Views

Also improved the `Find-DbaTrigger` to include triggers on views.

Motivation: https://twitter.com/TheRockstarDBA/status/1129059763373912065?s=01

### Commands to test
`Get-DbaDbObjectTrigger`
`Find-DbaTrigger`

### CODE REVIEW

File: functions/Get-DbaDbObjectTrigger.ps1 [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                    Stop-Function -Message "InputObject $_ not of type Table or View." -Continue
+                    return
+                }
+            }
+        }
+    }
+    process {
+        foreach ($Instance in $SqlInstance) {
+            $InputObject += Get-DbaDbTable -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Database $Database -ExcludeDatabase $ExcludeDatabase
+            $InputObject += Get-DbaDbView -SqlInstance $SqlInstance -SqlCredential $SqlCredential -Database $Database -ExcludeDatabase $ExcludeDatabase
```
Comment (wsmelton) [inline functions/Get-DbaDbObjectTrigger.ps1]: With the number of tables and views that can be in data warehouse systems I think we should offer up the ability to only retrieve one or the other. `Get-DbaDbObjectTrigger ...-Type Table` or `-Type View` ?

File: functions/Get-DbaDbObjectTrigger.ps1 [hunk 2]
```diff
@@ -0,0 +1,97 @@
+function Get-DbaDbObjectTrigger {
+    <#
+    .SYNOPSIS
+        Get all existing database triggers on one or more SQL instances.
```
Comment (wsmelton) [inline functions/Get-DbaDbObjectTrigger.ps1]: Need to update references to `database triggers` to be `object-level triggers`

File: functions/Get-DbaDbObjectTrigger.ps1 (hunk 1 above)
Comment (sqllensman) [reply functions/Get-DbaDbObjectTrigger.ps1]: An alternative would be to accept piping from Get-DbaDbTable or Get-DbaDbView

Comment (sqllensman) [reply functions/Get-DbaDbObjectTrigger.ps1]: An alternative would be to accept piping from Get-DbaDbTable or Get-DbaDbView

File: functions/Get-DbaDbObjectTrigger.ps1 (hunk 2 above)
Comment (ClaudioESSilva) [reply functions/Get-DbaDbObjectTrigger.ps1]: Thanks Shawn! Will update tonight.

File: functions/Get-DbaDbObjectTrigger.ps1 (hunk 1 above)
Comment (ClaudioESSilva) [reply functions/Get-DbaDbObjectTrigger.ps1]: Thanks Shawn & Patrick. Will definitely add the `-Type` identical to `Find-DbaTrigger` and see the piping from Get-*Table/View.

**Comment by ClaudioESSilva** [comment]
@potatoqualitee @wsmelton @sqllensman 
Seems it is stuck on appveyor/branch.
Regarding the failed ones there, I haven't change anything. 
What should I do? Will you force the merge?

**Comment by potatoqualitee** [comment]
mmm you will need to recreate your branches, it's super messed up - 1283 files modified. Please copy your changed files to a diff directory, get latest prerelease, rebranch then try again. Going to close, please feel free to reopen this PR if you can fix it, otherwise, I will wait for the updated PR.

**Comment by wsmelton** [comment]
There is only 3 changes for the PR.

**Comment by potatoqualitee** [comment]
my last commit must have worked but not showed up, thanks for rechecking 💯  will merge when tests pass

**Comment by wsmelton** [comment]
The branch failure is n/a as the PR test was successful. Since it is reopened it will have to run both again.

**Comment by potatoqualitee** [comment]
thanks so much for the new function! 🙇  gonna do a mass function comparison towards the end and add all new commands at once to psd1/psm1

**Comment by potatoqualitee** [comment]
boooom 💣

**Pull request merged**